tests/testthat/test-ppc-discrete.R

```diff
@@ -176,5 +177,30 @@ test_that("ppc_rootogram renders correctly", {
   vdiffr::expect_doppelganger(
     title = "ppc_rootogram (style='hanging', prob, size)",
     fig = p_custom_hanging)
+
+  p_discrete <- ppc_rootogram(
+    y = vdiff_y2,
+    yrep = vdiff_yrep2,
+    prob = 0.5,
+    size = 1,
+    style = "discrete"
+  )
+
+  vdiffr::expect_doppelganger(
+    title = "ppc_rootogram (style='discrete', prob, size)",
+    fig = p_discrete)
+
+  p_discrete_nonbound <- ppc_rootogram(
+    y = vdiff_y2,
+    yrep = vdiff_yrep2,
+    prob = 0.8,
+    size = 1,
+    style = "discrete",
+    bound_distinct = FALSE
+  )
+
+  vdiffr::expect_doppelganger(
+    title = "ppc_rootogram (style='discrete', prob, size, bound_distinct=FALSE)",
```

Comment:
We're getting a NOTE from R cmd check that the file path to the SVG that results from this is too long: 

https://github.com/stan-dev/bayesplot/actions/runs/17111544388/job/48533894818?pr=362#step:6:76

Maybe removing `style=` will make it short enough? E.g. `ppc_rootogram('discrete',...`